app/views/govuk_component/analytics_meta_tags.raw.html.erb

```diff
@@ -101,6 +101,10 @@
   end
 
   meta_tags["govuk:static-analytics:strip-postcodes"] = "true" if should_strip_postcode_pii?(content_item_hash, local_assigns)
+
+  stepnavs = links_hash[:part_of_step_navs] || []
+  stepnavs_content = stepnavs.map { |stepnav| stepnav[:content_id] }.join(",")
```

Comment:
I'm not quite sure how to pass several content IDs because our performance analyst, Tim, suggested to use a space instead of a comma to separate them so that CSVs are easier to handle. But we're using commas in other places, so I rather opted for being consistent.

Reply:
👍